DigitalLearningSolutions.Web/Controllers/LoginController.cs

```diff
@@ -69,50 +71,85 @@ public IActionResult Index(LoginViewModel model)
                     model.Password);
 
             var availableCentres = userService.GetUserCentres(verifiedAdminUser, approvedDelegateUsers);
+            (AdminClaimDetails? adminClaimDetails, List<DelegateClaimDetails> delegateClaimDetails) =
+                GetClaimsDetails(verifiedAdminUser, approvedDelegateUsers);
             if (availableCentres.Count == 1)
             {
-                if (verifiedAdminUser != null)
+                if (adminClaimDetails != null)
```

Comment:
Maybe we could refactor this into StartAdminSession(adminLoginDetails), which checks for null - then we can use it in Choose a centre too?

Reply:
I think I agree but I am not certain. If the null check was put inside StartAdminSession, then the code in here looks like we always start an admin session (though admittedly it would be with an adminDetails?.Id, so that might make it obvious that there are null checks inside)